src/main/java/thePackmaster/powers/showmanpack/HallOfMirrorsPower.java

```diff
@@ -0,0 +1,66 @@
+package thePackmaster.powers.showmanpack;
+
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.TextureAtlas;
+import com.evacipated.cardcrawl.mod.stslib.powers.abstracts.TwoAmountPower;
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInHandAction;
+import com.megacrit.cardcrawl.cards.green.AfterImage;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.localization.PowerStrings;
+import thePackmaster.SpireAnniversary5Mod;
+import thePackmaster.util.TexLoader;
+
+public class HallOfMirrorsPower extends TwoAmountPower {
+    public static final String POWER_ID = SpireAnniversary5Mod.makeID("HallOfMirrorsPower");
+    private static final PowerStrings powerStrings = CardCrawlGame.languagePack.getPowerStrings(POWER_ID);
+    public static final String NAME = powerStrings.NAME;
+    public static final String[] DESCRIPTIONS = powerStrings.DESCRIPTIONS;
+
+    public HallOfMirrorsPower(AbstractCreature owner, int amount) {
```

Comment:
Why do you not call the super ctor in here? Don't see a reason to repeat all this since AbstractPackmasterPower already supports 2 amount powers